applications/GeoMechanicsApplication/tests/dsettlement/fully_saturated_column_uniform_load/common/ProjectParameters_stage1.json

```
@@ -1,7 +1,7 @@
 {
   "problem_data": {
     "problem_name"                    : "mesh",
-    "start_time"                      : -0.001,
+    "start_time"                      : -1.0,
     "end_time"                        : 0,
```

Nitpicking: shouldn't we use floating-point notation here too?
```suggestion
    "end_time"                        : 0.0,

---

Done